capstone/src/main/java/com/step902020/capstone/EventController.java

```diff
@@ -52,18 +52,17 @@ public RedirectView saveEvent (
      @RequestParam("requiredFee") Optional<Boolean> requiredFee,
      @RequestParam("event-id") String eventId
     ) throws IOException {
-      Organization organization = organizationRepository.findByEmail(user.getEmail()).get(0);
+      Organization organization = organizationRepository.findByEmail(user.getEmail()).orElse(null);
       Event event = eventId.length() <= 0? null : this.eventRepository.findById(Long.parseLong(eventId)).orElse(null);
       if (event != null) {
         event.setEventDateTime(eventDateTime);
         event.setEventDescription(eventDescription);
         event.setEventLatitude(Double.parseDouble(eventLatitude));
         event.setEventLongitude(Double.parseDouble(eventLongitude));
         event.setEventTitle(eventTitle);
```

Comment:
(just a heads up) There could be a potential error here involving data retrieval and the hardcoded filters I added. I plan on looking into it but got lost with maps lol (there was something about the Optional<boolean> type and getting the filter that java didn't like). Likely something to look into sooner rather than later

Reply:
If you want to see this locally, make setters for the hardcoded filters to see the compile error I got before

Reply:
created issue #103; I'll fix it

Reply:
I did not get any compile error with the setters, so there may not be an issue after all?